sharq/queue.py

```diff
@@ -452,15 +452,15 @@ def clear_queue(self, queue_type=None, queue_id=None, purge_all=False):
             raise BadArgumentException('`queue_type` has an invalid value.')
 
         response = {
-            'status': 'Failure',
+            'status': 'failure',
             'message': 'No queued calls found'
             }
         # remove from the primary sorted set
         primary_set = '{}:{}'.format(self._key_prefix, queue_type)
         queued_status = self._r.zrem(primary_set, queue_id)
         if queued_status:
-            response.update({'status': 'Success', 
-                    'message': 'Successfully removed all queued calls'})
+            response.update({'status': 'success',
```

Comment:
IS this the api response? If yes, can we make sure clients don't break because of this change

Reply:
Hi Tarun,

These are the latest feature addition to sharQ service, and only voice is using it, this won't break anything as we are only relaying the message back.

Reply:
this won't break anything.